src/lib/flow/Kconfig

```diff
@@ -13,7 +13,7 @@ config NODE_DESCRIPTION
 
 config RESOLVER_CONFFILE
 	bool "Resolver conffile"
-	depends on NODE_DESCRIPTION
+	depends on NODE_DESCRIPTION && HAVE_GLIB
```

Comment:
Why have the build broken until then? The patch should fix resolver-conffile and remove this line. Isn't an one line patch and is breaking galileo build that is being used by QA